internal/db/schema/migrations/oss/postgres/100/02_iam_role_org.up.sql

```diff
@@ -23,6 +23,24 @@ begin;
     ('children'),
     ('individual');       
 
+
+  create function delete_org_role_individual_role_grant_scope() returns trigger
+  as $$
+  begin
+    if new.grant_scope = 'children' then
+      delete from iam_role_org_individual_grant_scope
+        where role_id = new.public_id;
+    end if;
+    return new;
+  end;
+  $$ language plpgsql;
+  comment on function delete_org_role_individual_role_grant_scope() is
+    'delete_org_role_individual_role_grant_scope deletes individual role grants from '
+    'iam_role_org_individual_grant_scope to remove redundant grants';
```

Comment:
Same with this function, I think the repository should handle this, and this should be in a separate PR.